index.html

```diff
@@ -19,7 +25,137 @@
+                <div class="col-sm-4 col-md-12">
+                  <img class="col-xs-12" src="static/img/feature-1.jpg" />
+                </div>
+                <div class="caption col-sm-8 col-md-12">
+                  <h2>
+                    Free to Use - No Registration Required
+                  </h2>
+                  <p class="lead">
+                    Betta is a free, open source web application created by <a href="http://www.dabapps.com" target="_blank" title="DabApps.com">DabApps</a>. You don't need to register, like us on Facebook, or any of that.
+                  </p>
+                  <p class="lead">
+                    Just jump straight into the app and <a class="start-app" title="Start Using Betta">start creating</a>!
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
 
+          <div class="col-md-4">
+            <div class="thumbnail">
+              <div class="row">
+                <div class="col-sm-4 col-md-12 col-sm-push-8 col-md-push-0">
+                  <img class="col-xs-12" src="static/img/feature-2.jpg" />
+                </div>
+                <div class="caption col-sm-8 col-md-12 col-sm-pull-4 col-md-pull-0">
+                  <h2>
+                    Quick and Easy to Use
+                  </h2>
+                  <p class="lead">
+                    Betta takes Bootstrap's variables and gives you an easy to use interface, allowing you to quickly create unique styles without writing any less / css yourself.
+                  </p>
+                  <p class="lead">
+                    With our instant previews, minimal interface, quick search feature &amp; color picker, you'll have a shiny new design in no time.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="col-md-4">
+            <div class="thumbnail">
+              <div class="row">
+                <div class="col-sm-4 col-md-12">
+                  <img class="col-xs-12" src="static/img/feature-3.jpg" />
+                </div>
+                <div class="caption col-sm-8 col-md-12">
+                  <h2>
+                    Import &amp; Export
+                  </h2>
+                  <p class="lead">
+                    Import variables from an existing project, or export variables created with Betta.
+                  </p>
+                  <p class="lead">
+                    There are options for importing &amp; exporting variables that allow you to easily merge styles from multiple projects and only export the things you care about.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </section>
+
+    <section class="section">
+
+      <div class="container-fluid footer">
+        <div class="container">
+          <div class="row">
+            <div class="col-md-6">
```

Comment:
These could probably be col-sm-6 and still fit. Or this one slightly larger on small screens.